### PR TITLE
Update VS Code Java Testing doc to reflect new Testing API from VS Code Core

### PR DESCRIPTION
**Pull request by nickzhums**

### CODE REVIEW

**Comment by nickzhums** [comment]
@gregvanl could you help review this PR, thanks

File: docs/java/java-testing.md [hunk 1]
```diff
@@ -11,132 +11,126 @@ MetaSocialImage:
...
+If you are interested in providing feedback or contributing directly to the code base, please check the document [Contributing to Java Test Runner](https://github.com/Microsoft/vscode-java-test/blob/main/CONTRIBUTING.md), which covers the following parts:
+- [Questions and Feedback](https://github.com/Microsoft/vscode-java-test/blob/main/CONTRIBUTING.md#questions-and-feedback)
+- [Reporting Issues](https://github.com/Microsoft/vscode-java-test/blob/main/CONTRIBUTING.md#reporting-issues)
+- [Contributing Fixes](https://github.com/Microsoft/vscode-java-test/blob/main/CONTRIBUTING.md#contributing-fixes)
 
-## More Information
 
-Visit the [GitHub repository](https://github.com/microsoft/vscode-java-test) of the [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) for more details on commands and settings.
 
-## Next steps
```
Comment (jdneo) [inline docs/java/java-testing.md:137]: I think we should reserve the `Next Steps` section

**Comment by gregvanl** [comment]
Hi @nickzhums Looks good. I did an edit/format pass in https://github.com/microsoft/vscode-docs/commit/ca0c34a9ad0096f55fb705422cdfb6d16ae26abd.
One reminder: if you are creating or updating images, please enable Git LFS for binary files per https://github.com/microsoft/vscode-docs/blob/main/README.md#cloning to help keep the repo size manageable.

**Comment by nickzhums** [comment]
> Hi @nickzhums Looks good. I did an edit/format pass in [ca0c34a](https://github.com/microsoft/vscode-docs/commit/ca0c34a9ad0096f55fb705422cdfb6d16ae26abd).
> One reminder: if you are creating or updating images, please enable Git LFS for binary files per https://github.com/microsoft/vscode-docs/blob/main/README.md#cloning to help keep the repo size manageable.

got it, thanks!

**Pull request merged**